go/vt/vttablet/tabletmanager/vreplication/vcopier.go

```diff
@@ -669,6 +669,16 @@ func (vc *vcopier) copyTable(ctx context.Context, tableName string, copyState ma
 	return nil
 }
 
+func (vc *vcopier) updatePos(ctx context.Context, gtid string) error {
```

Comment:
Worth adding a comment to this function on when it might be called, what's its use case.

Reply:
Added comment.